docs/src/reference/commandline.md

```diff
@@ -369,8 +386,14 @@ Arguments:
   [NAMESPACES]...  The namespaces that are generated by `generate jsi bindings`
 
 Options:
-      --config <CONFIG>  The configuration file for this build
-  -h, --help             Print help
+      --config <CONFIG>  
+          The configuration file for this build
+
+      --native-bindings 
+          This will add required implementations to the generated `build.gradle` file.
```

Comment:
As written, it might be easy to misunderstand that this is only required when also generating native bindings.
 
```suggestion
          This will add implementations required for native Android bindings to the generated `build.gradle` file.
```